test/test-applications/integrations/Samples.WebRequest/RequestHelpers.cs

```diff
@@ -318,7 +318,7 @@ public static async Task SendWebRequestRequests(bool tracingDisabled, string url
                 using (Tracer.Instance.StartActive("GetResponse"))
```

Comment:
I just realized the test isn't testing the new `GetRequestStream` integration. Can you add the usage here? This would also be a good way to test the scenario where different threads call `GetRequestStream` and `GetResponse`

Reply:
Oh. I started by writing the tests, and it failed for `UploadData` because it uses `GetRequestStream` internally, so the method is actually tested. But you're right that I should make it explicit, gonna add one more case 👍

Reply:
Ah I see, I didn't realize that. Thank you for adding another test case 😄